### DISCUSSION TITLE
Changing the database on Mongo

### DISCUSSION THREAD

**Discussion by achaconm2001** [prisma/prisma#11688] [Q&A] [answered]
At the documentation of the new feature raw queries for MongoDB, it says that "The raw query API for MongoDB differs from Prisma's $queryRaw SQL API to handle some low-level differences between databases to give you a better API and developer experience."

I could not find a way to change the current database of use the getDb method from the raw mongo api.
Did I misunderstand the comment or am I missing something?

**Comment by dpetrick** [comment, marked as answer]
I can't find the specific docs entry you're mentioning, but if it's like that in the docs it's not ideal. The raw API is there to provide users a way to express queries that Prisma may not yet be able to express. Mind you that the raw API is for the exception of `find` and `aggregate` exactly like it's described in the Mongo command reference: https://docs.mongodb.com/manual/reference/command/

I can't find `getDb` in there. I assume you mean the Mongo shell / client method `Mongo.getDb` - that's _not_ what the raw API is. The raw API is not a Mongo shell or Mongo Node client (or similar) replacement. The Mongo shell and client build on top of the raw commands that you see in the links above to offer the interface they do, but we're not offering a full client with this API.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
